### PR TITLE
[platform] Add as7512 sfp/eeprom utils

### PR DESCRIPTION
**Pull request by NStetskovych**

### CODE REVIEW

File: files/build_templates/swss.service.j2 [hunk 1]
```diff
@@ -40,6 +41,7 @@ ExecStopPost=/usr/bin/mst stop
 {% elif sonic_asic_platform == 'cavium' %}
 ExecStopPost=/etc/init.d/xpnet.sh stop
 ExecStopPost=/etc/init.d/xpnet.sh start
+ExecStopPost=/usr/bin/accton_as7512_util.py clean
```
Comment (lguohan) [inline files/build_templates/swss.service.j2]: is this asic related or peripheral related? usually, this only contains asic related init.

**Comment by lguohan** [comment]
@jleveque to have another look.

**Comment by jleveque** [comment]
Looks good to me, but I will wait to approve until your question regarding files/build_templates/swss.service.j2 above is settled.

File: files/build_templates/swss.service.j2 (hunk 1 above)
Comment (NStetskovych) [inline files/build_templates/swss.service.j2]: it is peripheral related

Comment (lguohan) [inline files/build_templates/swss.service.j2]: in this case, can it be moved to the the as7512 platform package instead of here?

**Comment by NStetskovych** [comment]
@lguohan done.

**Comment by NStetskovych** [comment]
@lguohan pls review
